### ISSUE TITLE
cannot access attribute AND value at the same time for the element

### ISSUE THREAD

**Issue by aedna** [databricks/spark-xml#340]
using spark-xml 0.1.1-s_2.11 on Apache Spark 2.3.1, Scala 2.11

I uploaded a sample XML from github site:

```
<one>
     <two myTwoAttrib="BBBBB">two</two>
      <three>three</three>
</one>
```
and the schema produced doesn't include the attribute:

I get:

```
root
|-- one: struct (nullable = true)
|    |-- three: string (nullable = true)
|    |-- two: string (nullable = true)

```
------------------

When I create my own schema:

```
customSchema = StructType([
  StructField("one",
              StructType([
                StructField("three",StringType(),True),                
                StructField("two",StructType([
                          StructField("myTwoAttrib",StringType(),True),
                          StructField("_VALUE", StringType(),True),
                        ]), True)
              ]),True)])
```

I get the attribute, but cannot get the value "two" anymore

So I am able to get the attribute OR the value but not both at the same time

I am using following options:

`option("valueTag","_VALUE").option("excludeAttribute","false").option("attributePrefix","_")`

Can you help please?

Thanks a lot in advance!

**Comment by thecolorkeo** [comment]
I believe I am having this issue too:
<text xml:space="preserve" bytes="390">Large city in the state of...</text>

df.select('text')

given input columns: [parentid, id, minor, comment, text, model, format, contributor, timestamp, sha1]

It runs without errors, but doesn't print anything for "text" field. So I'm unable to access the string in <text>. I also tried inferred and custom schema.

**Comment by mliszewski** [comment]
Is there any workaround/fix for this? I am still having trouble accessing both an attribute and a value for the same element.

**Comment by srowen** [comment]
I can't reproduce this on 0.6.0. It may be something long since fixed (this reported vs 0.1.1?)

```
    val df = spark.read
      .option("rowTag", "one")
      .option("inferSchema", true)
      .xml("/tmp/test.xml")
    df.show()
    df.printSchema()

...

+-----+------------+
|three|         two|
+-----+------------+
|three|[two, BBBBB]|
+-----+------------+

root
 |-- three: string (nullable = true)
 |-- two: struct (nullable = true)
 |    |-- _VALUE: string (nullable = true)
 |    |-- _myTwoAttrib: string (nullable = true)
```

If there's a current reproduction, let me know.
